Sources/Errors/UnableToLoginErrorViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import GDSAnalytics
+import GDSCommon
+import Logging
+
+struct UnableToLoginErrorViewModel: GDSErrorViewModel, BaseViewModel {
+    let image: String = "exclamationmark.circle"
```

Comment:
Would be good to put TODO's wherever you've got strings that will need localisation. There's an ever increasing piece of tech debt here

Reply:
fair do's 🙂